src/templates/TemplateData.ts

```diff
@@ -243,8 +255,9 @@ export function removeLanguageFromId(id: string): string {
 }
 
 // tslint:disable-next-line:no-unsafe-any
-async function downloadAndExtractZip(templateUrl: string, release: string): Promise<{}> {
+async function downloadAndExtractZip(cliFeedJson: cliFeedJsonResponse, release: string): Promise<{}> {
```

Comment:
nit: Can you rename this function? It bothers me that it's a generic name that should work with literally any zip file I want to download, but the implementation only works for the templates zip file. How about `downloadAndExtractTemplates` or just `downloadTemplates`